x-pack/platform/plugins/shared/onechat/public/plugin.tsx

```diff
@@ -64,12 +66,14 @@ export class OnechatPlugin
     const chatService = new ChatService({ http });
     const conversationsService = new ConversationsService({ http });
     const toolsService = new ToolsService({ http });
+    const dataService = new DataService({ http });
 
     this.internalServices = {
       agentService,
       chatService,
       conversationsService,
       toolsService,
+      dataService,
```

Comment:
what about `DataTypeComponentRegistry` 

we defined this component registry but I don't see this instantiated anywhere? This allows other plugins to register frontend components (e.g. configuration screen)

I'm thinking about something like that:
https://github.com/elastic/kibana/blob/main/x-pack/solutions/chat/plugins/workchat-app/public/plugin.tsx#L78